src/scenes/Question/scenes/Read/components/Meta/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { DateTime } from 'luxon'
+
+import IconMenu from 'react-toolbox/lib/menu/IconMenu'
+import MenuItem from 'react-toolbox/lib/menu/MenuItem'
+
+import Flag from '../Flag'
+
+class Meta extends Component {
+  render () {
+    const { node, createFlag } = this.props
+
+    const flagIcons = {
+      outdated: 'history',
+      incomplete: 'report'
+    }
+
+    const flags = node.flags.map((flag, i) => {
+      return (
+        <Flag
+          key={i}
+          caption={flag.type}
+          icon={flagIcons[flag.type]}
+          tooltip={
+            'By ' +
+            flag.user.name +
+            ' on ' +
+            DateTime.fromISO(flag.createdAt).toFormat('dd LLL yyyy')
+          }
+        />
+      )
+    })
+
+    return (
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignItems: 'center'
+        }}
+      >
+        <b>
+          Answered by {node.answer.user.name} on{' '}
+          {DateTime.fromISO(node.answer.updatedAt).toFormat('dd LLL yyyy')}:
+        </b>
+        <div
+          style={{
+            display: 'flex',
+            alignItems: 'center'
+          }}
+        >
+          {flags}
+          <IconMenu icon="flag">
+            {Object.keys(flagIcons).map((type, i) => {
```

Comment:
You could use `map` from `lodash` that handle Arrays and Collections

Reply:
Oh thanks, I was looking for something like this